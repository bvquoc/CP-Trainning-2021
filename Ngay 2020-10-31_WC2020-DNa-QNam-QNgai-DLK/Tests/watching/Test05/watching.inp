100 1000 800000 600
4691 19603 23526 28288 46610 48323 48924 49525 55821 72629 81447 82741 86320 93237 106910 110939 137153 138862 160308 160909 161510 163247 167736 180531 205105 220857 223452 228303 228904 258642 268869 270779 279552 292257 293105 323119 323720 336772 338473 349910 350831 372679 378873 381324 382161 387208 392620 393221 401825 402532 410264 410943 411950 413845 415229 420193 422039 426616 453675 454517 467176 469037 478026 482907 491476 494178 506444 512003 517695 539837 555700 557034 564291 571214 575481 579559 595728 601349 605167 617948 627012 640259 640860 646312 650333 653135 654145 674063 678762 687361 688345 695833 719556 726684 736901 755777 756378 772326 787328 789542
1495 3583 4184 4785 5386 5987 6588 7189 7790 8391 8992 9593 10194 10795 11396 11997 12598 13199 13800 14401 15002 15603 16204 16805 17406 18007 18608 20938 21632 23554 24318 26110 27741 28796 29397 29998 31553 32154 34051 34652 35253 35854 36455 37056 38266 39960 40561 41162 44688 46772 49016 49617 50770 51371 51972 53575 54220 54821 55422 56023 56624 57225 57826 58427 59028 59629 60230 60831 61432 62033 62716 63317 63918 64519 65120 65721 66322 66923 67524 68125 68726 69327 69928 70529 71130 71731 72332 72933 73534 74135 74736 75337 75938 76539 77140 77741 78342 79046 79722 80323 80924 81525 82126 82727 83328 83929 84530 85284 87464 88307 88908 89509 90110 90711 91312 92026 92627 93228 94123 94835 96694 97985 98586 99187 99788 100389 100990 101591 102192 102793 103394 103995 104596 105197 105798 106638 107239 107840 108441 109042 110772 111373 111974 112575 113271 114717 115318 115919 116520 117121 117722 118323 118924 119525 120126 120727 121328 121929 122820 123421 124082 124683 125284 126824 127425 128026 128627 129228 131108 131709 132696 133380 134571 135172 136127 141490 142091 142692 143587 144188 144828 145590 146191 148723 149331 149971 150572 151173 151774 152375 152976 154356 154957 155558 156159 156760 157361 157962 158563 159164 159765 160366 160967 161568 162169 162770 163371 163972 164573 165174 165775 166376 167590 168356 168957 171213 171814 172415 173451 174096 176621 177222 177823 180200 183302 183903 184832 185433 187747 189546 190533 191134 191735 192766 197693 198996 199597 200198 200799 201400 202001 202602 203203 203804 204405 205006 205607 206208 206809 207410 208011 208750 209427 210161 212035 213802 214403 215004 215605 216206 216807 217985 218586 219187 221795 222396 223472 224416 225017 227850 228451 229052 229653 230254 230855 231456 232057 232658 233259 233860 234461 235062 235663 236264 236865 237466 238067 240103 240704 241305 242606 243207 245302 245903 246962 248732 249333 249934 250554 251351 252010 252611 253773 254374 254975 257718 258494 259095 260463 261064 261665 262539 263140 263741 264342 264943 266009 266992 267593 268194 268795 269396 269997 270598 272273 274736 277703 278304 278905 279506 280107 280708 281309 281910 282511 283112 283713 284314 284915 285547 286459 287329 287930 288531 289173 290138 290739 291340 291941 292542 293143 294488 295169 295986 296587 297188 297959 298785 299937 302934 303535 304136 304737 305596 306280 307991 309305 309906 311002 311603 312301 312902 313503 314104 314705 315306 315907 316508 317198 318619 319220 319945 320546 321147 321748 322349 322950 323551 324152 324753 325822 326423 327024 327625 328226 328827 329428 330029 330630 331898 332531 333180 333781 334382 334983 335584 336185 336786 337387 337988 338589 341479 342712 343313 343914 345182 345783 348454 349469 350070 351719 352320 352921 353522 354516 355141 357440 358041 359227 359901 360502 361103 361704 362305 362906 363507 364108 364709 365310 365911 366512 367113 367714 368315 368916 369517 370118 370719 373078 373679 374280 374881 375709 377990 378591 379192 379966 381941 382542 384192 384793 385394 388490 392795 393396 394139 394740 395341 396823 397424 398025 398626 399227 399828 400429 401030 401631 402232 402833 403434 404035 404636 405237 405838 406439 407040 407641 408242 408843 409444 410045 410992 413637 414238 414971 415768 416846 417447 418048 418671 419381 419982 421737 422864 424338 424939 426267 427088 427689 428290 428891 430204 431087 431688 432428 433257 433858 435642 436243 436844 437445 438046 438647 439248 439849 440450 441051 441652 442253 442854 443455 444056 444657 445258 445859 446460 447061 447662 448263 448864 449465 450066 450667 451268 451869 452470 453071 453672 454273 454874 455475 456076 456677 457278 457879 458480 459081 459682 460283 460884 461485 462086 462687 463288 463889 464490 465091 465692 466293 466894 467495 468096 468697 469298 469899 470500 471101 471702 472303 472904 473505 474106 474707 475308 477564 479411 480012 481190 482614 483215 483816 484417 485018 485619 486220 486821 487422 488023 488624 489225 489826 490427 491028 491629 492230 492831 493432 494033 494634 495235 495836 496437 497038 497639 498240 498841 499442 500043 500644 501245 501846 502447 503048 503649 504250 504851 505452 506053 506654 507255 507856 508457 509058 509659 510260 510861 511462 512257 513375 514731 515622 516798 517399 518588 519189 519790 520762 522453 523727 524669 526174 526899 527500 528101 528702 529303 529904 530505 531106 531707 532308 532909 533510 534111 534712 535313 535914 536515 537116 537717 538318 538919 539520 540121 540722 541323 541924 542525 543126 543727 544328 544929 545530 546131 546732 547333 547934 548535 549136 549737 550338 551122 553244 553845 554446 556343 557026 557627 558345 558946 559547 560148 560749 561509 562110 562711 563413 565015 565616 566217 566818 567419 568020 568621 569222 569823 570424 571025 571626 572227 572828 573429 574030 574631 575232 575833 577010 577611 578212 579760 581206 583229 584414 585015 585616 586500 587101 587740 589624 590762 591363 591964 594718 595319 595920 596521 597122 597723 598324 598925 599526 600127 600728 601329 601930 602531 605692 606293 607801 608508 609109 612994 613595 614196 614797 615398 615999 616600 617201 617802 618403 619004 619605 620206 620807 621408 622009 622610 623211 623812 624413 625014 625615 626216 626817 627418 628304 628905 629506 630107 630708 631309 631910 632511 633112 633713 634314 634915 635516 636117 636718 637319 638230 641203 641804 642405 643006 643856 645303 645904 646505 647106 647707 648308 648909 649510 651311 651912 652513 653114 653715 654540 655927 656528 657129 657828 658429 659030 659631 660232 660833 661434 662035 662636 663237 663838 664439 665040 665641 666242 666843 667444 668045 668646 669247 669848 670449 671050 671651 672252 672853 673454 674055 674656 675257 675858 676459 677060 677661 678262 678863 679464 680065 680666 681267 681868 682469 683070 684236 684837 685438 689709 691398 692097 692698 693299 693900 694501 695102 695703 696304 696905 697506 699227 699828 701719 702320 702921 703954 704555 705156 705757 706358 706959 707560 708161 708762 709363 709964 710565 711166 711767 712368 712969 714110 715998 716599 718593 719303 721408 722269 723567 724557 725343 725944 726545 727146 727747 728348 728949 729550 730151 730752 732133 732734 733335 735314 735915 736516 737117 737882 738483 739084 739685 740286 740887 741488 742089 742690 743291 743892 744493 745094 745695 746296 746897 747498 748330 749393 750169 750770 751371 751972 752573 754147 755519 756217 756818 757419 758508 759109 760319 760920 762026 763513 764178 764779 765380 765981 766582 768162 768763 769364 771057 771658 772259 774950 775651 776252 776853 777454 778874 779713 782191 782792 783393 783994 785742 786343 786944 787545 788554 789155 789756 792865 793466 794067 795974 796688 799408
