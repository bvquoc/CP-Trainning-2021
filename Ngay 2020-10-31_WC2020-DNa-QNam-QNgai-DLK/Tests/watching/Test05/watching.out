722504 33015
